index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"></link>
```

Comment:
- [OPTIONAL] Great work adding the syles link as requested by the previous reviewer, I think, even though it works well now, you should avoid ending with a closing tag a `<link rel="stylesheet" href="style.css" />` is enough, I think this would make your project look more professional.